projects/coresky/index.js

```diff
@@ -0,0 +1,13 @@
+const { staking } = require("../helper/staking");
+const { pool2 } = require("../helper/pool2");
+
+const coresky_ETHEREUM = "0x179886f229f9cd52016c6dbf4be66e18111ec6ab";
```

Comment:
this does not look like an erc20 token address, also dont see any tokens staked in the controller contract